projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -229,4 +229,15 @@ public function add_options_menu( $wp_admin = false ) {
 
 		parent::add_options_menu( $wp_admin );
 	}
+
+	/**
+	 * Adds Appearance menu.
+	 *
+	 * @param bool $wp_admin Optional. Whether links should point to Calypso or wp-admin. Default false (Calypso).
+	 * @param string $customize_slug. Whether customizer links should point to Calypso or wp-admin.
+	 */
+	public function add_appearance_menu( $wp_admin = false, $customize_slug ) {
+		$customize_slug = 'customize.php';
+		parent::add_appearance_menu( $wp_admin, $customize_slug );
```

Comment:
For the purposes of this PR I think this is ok. In general however, I think it points to a potential issue with the code being overly reliant on a class hierarchy.

If we were to decouple this and use traits to "mixin" the functionality required into each class we could call `add_appearance_menu` directly and pass the parameters once, allowing us to avoid this polymorphic approach.

I've spoken to @mmtr about this as well as a potential avenue for refactoring.

Reply:
I am not fan of this either. I will fix the notices and let's refactor the approach on a new PR just to allow @jeherve to cherry pick this on Jetpack 9.4